.travis.yml

```diff
@@ -16,6 +16,7 @@ branches:
 # install package and dependencies
 install:
   - pip install coverage nose pytest pytest-cov
+  - pip install scikit-learn
```

Comment:
Why was this dependency added?

Reply:
Yes, I wanted to talk to you about this.
This is not a dependency for the codes, but if this is added, Travis actually tests all the OWL norm codes. This bug was missed especially as scikit learn was not installed and none of OWL norm tests ran.
scikit-learn in itself will continue to remain an optional dependency